dashboard/models.py

```diff
@@ -604,6 +605,7 @@ class Timepoint(db.Model):
             nullable=False)
     is_phantom = db.Column('is_phantom', db.Boolean, nullable=False,
             default=False)
+    header_diffs = db.Column('header_diffs', JSONB)
     # These columns should be removed when the static QC pages are made obsolete
     last_qc_repeat_generated =  db.Column('last_qc_generated', db.Integer,
             nullable=False, default=1)
```

Comment:
`def study()` is duplicated at line 632 and 678

Reply:
Wow. How did I end up doing that? Must have been a merge gone wrong hahaha. Thanks for catching it